repos.json

```diff
@@ -57,6 +58,7 @@
     { "language": "java", "repo": "googleapis/java-phishingprotection", "apiHint": "phishingprotection"},
     { "language": "java", "repo": "googleapis/java-pubsub", "apiHint": "pubsub"},
     { "language": "java", "repo": "googleapis/java-recaptchaenterprise", "apiHint": "recaptcha"},
+    { "language": "java", "repo": "googleapis/java-recommender", "apiHint": "recommender"},
```

Comment:
This is an AI thing, right?  Thinking we should update `teams.json` along with this PR to make sure attribution for issues is correct

Reply:
Yep, SGTM.

Reply:
Done. Added recommender to the animl team